app/src/main/java/com/mobiledevelopmentfoodapp/service/RestaurantService.kt

```diff
@@ -9,9 +9,10 @@ import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
 class RestaurantService {
-    suspend fun fetchFoods(): List<Food>?{
-        return withContext(Dispatchers.IO){
-            val service = RetrofitClientInstance.retrofitInstance?.create(IFoodDAO::class.java)
+    suspend fun fetchFoods(): List<Food>? {
+        return withContext(Dispatchers.IO) {
```

Comment:
Added a space after the question mark and before the brackets according to https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace